firebase-firestore/src/main/java/com/google/firebase/firestore/Query.java

```diff
@@ -365,11 +369,12 @@ public Query whereArrayContainsAny(@NonNull FieldPath fieldPath, @NonNull List<O
 
   /**
    * Creates and returns a new Query with the additional filter that documents must contain the
-   * specified field and that the field must equal any values of the provided array.
+   * specified field and the value must equal at least one value from the provided array.
```

Comment:
"at least one value from" is odd phrasing for whereIn, since the only way it would match multiple values is if your array contains duplicate values, which would probably not be an intentional thing to do...  So I would just say "the value must equal one of the values from ..."